cloud/azure/README.md

```diff
@@ -0,0 +1,68 @@
+# SignalFx Azure Integration
+
+
+## Usage
+
+```hcl
+terraform {
+  required_version = ">= 0.12"
+  required_providers {
+    azurerm  = "~> 1.44"
+    azuread  = "~> 0.7"
+    signalfx = "~> 4.16"
+  }
+}
+
+provider "azurerm" {
+  subscription_id = var.azure_subscription_id
+  tenant_id       = var.azure_tenant_id
+}
+
+provider "azuread" {
+  subscription_id = var.azure_subscription_id
+  tenant_id       = var.azure_tenant_id
+}
+
+variable "sfx_token" {
+  type = string
+}
+
+provider "signalfx" {
+  auth_token = var.sfx_token
+  api_url    = "https://api.eu0.signalfx.com"
+}
+
+module "signalfx-integrations-cloud-azure" {
+  source                 = "git::ssh://git@github.com/claranet/terraform-signalfx-integrations.git//cloud/azure"
```

Comment:
I'd use the simpler github notation in the general example:
```suggestion
  source                 = "github.com/claranet/terraform-signalfx-integrations.git//cloud/azure"
```

Tested with:

```
  source                 = "github.com/claranet/terraform-signalfx-integrations.git//cloud/azure?ref=azure_integration"
```

Reply:
Best will be an Hashicorp Registry link ;)

Reply:
yeah this will be the hashicorp registry when ready we can ignore that for now ;)